app/views/JourneyRecoveryContinueView.scala.html

```diff
@@ -29,7 +29,7 @@ <h1 class="govuk-heading-l">@messages("journeyRecovery.continue.heading")</h1>
 
     <p class="govuk-body">
         @govukButton(
-            ButtonViewModel(messages("site.continue"))
+            ButtonViewModel(messages("site.startAgain"))
```

Comment:
Refer to pr description for any weirdness related to journey recovery. Or give me a message :)